unitTests/Simulate.ts

```diff
@@ -58,20 +58,30 @@ export class Simulate {
   }
 
   static queryError(env: IMockEnvironment, options?: ISimulateQueryData): ISimulateQueryData {
-    options = _.extend(
-      {},
-      {
+    const opt = {
+      ...options,
+      ...{
         error: new QueryError({
           statusCode: 500,
           data: {
             message: 'oh',
             type: 'no!'
           }
         })
-      },
-      options
-    );
-    return Simulate.query(env, options);
+      }
+    };
+
+    return Simulate.query(env, opt);
+  }
+
+  static noResults(env: IMockEnvironment, options?: ISimulateQueryData): ISimulateQueryData {
+    const opt = {
+      ...options,
+      ...{
+        results: FakeResults.createFakeResults(0)
```

Comment:
We could set `results` directly on line 80 rather than placing it inside an object and destructuring.
Same comment for line 63.